internal/pkg/service/appsproxy/dataapps/wakeup/wakeup.go

```diff
@@ -0,0 +1,76 @@
+// Package wakeup sends wakeup requests for an app, if the proxy received a request for the app, but the app does not run.
+// The first request is sent immediately, the next after the Interval, otherwise the request is skipped.
+package wakeup
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/benbjohnson/clock"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/appsproxy/dataapps/api"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/appsproxy/syncmap"
+)
+
+// Interval sets how often the proxy sends wakeup request to sandboxes service.
+// If the last notification for the app was less than this Interval ago then the notification is skipped.
+const Interval = time.Second
```

Comment:
I suggest to use some limit also there, to mitigate DoS attacks, and stop them on the proxy.